Automattic/jetpack
# VideoPress: Introduce minimal `Edit Details` page

Introduce the routes and minimal version of `Edit Details` page.

Closes #26036 

#### Changes proposed in this Pull Request:

* Add `EditDetails` page.
* Fix `ClipboardButtonInput` inside wordpress env.
* Add `background-color` for `VideoDetails`.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `VideoPress`
* Navigate to admin page and append `#/video/1/edit` into the URL
* You should see the mocked edit details page.

#### Demo

<img width="596" alt="image" src="https://user-images.githubusercontent.com/1663717/189421286-316a7efa-21e1-46a5-85b1-aa25f374211a.png">
